.github/workflows/visual_regression_tests.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   install-cache:
     name: Install and cache dependencies
-    if: github.triggering_actor != 'dependabot[bot]' || contains(github.event.pull_request.draft, false)
+    if: ${{ github.triggering_actor != 'dependabot[bot]' || contains(github.event.pull_request.draft, false) }}
```

Comment:
I don't think we need the `${{ }}`

Reply:
Also, why `contains` instead of `==`?

Reply:
I tried to use it without the `${{ }}` and it didn't word. :(

Reply:
`contains` or `==` it's the same. I saw this function here https://docs.github.com/en/actions/learn-github-actions/expressions#contains and wanted to use it. 🙏🏻